source/tool/chipsec/hal/acpi_tables.py

```diff
@@ -584,3 +584,27 @@ def __str__( self ):
 ACPI Table Entries:
 %s
 """ %( ''.join( ['0x%016X\n' % addr for addr in self.Entries]))
+
+########################################################################################################
+#
+# FACP Table
+#
+########################################################################################################
+
+class FACP (ACPI_TABLE):
+  def __init__( self ):
+    self.DSDT = None
+    self.X_DSDT = None
+
+  def parse( self, table_content ):
+    self.DSDT = struct.unpack('=I', table_content[4 : 8])

```

Comment:
It should be explicit little-endian '<'.  
No space around the ':' when slicing.